packages/integration-testsuite/src/apolloServerTests.ts

```diff
@@ -2777,24 +2776,6 @@ export function defineIntegrationTestSuiteApolloServerTests(
         );
       });
 
-      // We can remove this test after the initial AS4 release once we stop supporting
-      // the playground package.
-      it('can install playground with specific version', async () => {
```

Comment:
We could also leave the dependency and test in place, which would ensure that we don't accidentally break the ability to use the plugin. Thoughts @trevor-scheer?

Reply:
(ie, just loading the package from npm like a normal dependency)

Reply:
I like that, we'll at least be reminded we broke support for it if and when that happens.